src/tss2-tcti/tcti-device.c

```diff
@@ -393,7 +421,11 @@ Tss2_Tcti_Device_Init (
     memset (&tcti_common->header, 0, sizeof (tcti_common->header));
     tcti_common->locality = 3;
 
+#ifdef __VXWORKS__
+    tcti_dev->fd = open (dev_path, O_RDWR, 0666);
```

Comment:
Since we do not intent to create a file here (no O_CREAT), we should not set a file-access-mode.

Reply:
VxWorks will check the flags and mode. Only the fixed value can be accepted by TPM driver.

Reply:
> Only the fixed value can be accepted by TPM driver.
What do you mean by this ?

Also don't know about 0666 for the mode ? If it's ignored, then 0 would be good enough, right ?

Reply:
I updated the code for using macro to replace the magic number.